zingolib/src/lightclient/propose.rs

```diff
@@ -146,7 +146,6 @@ impl LightClient {
 
     /// Unstable function to expose the zip317 interface for development
     // TOdo: add correct functionality and doc comments / tests
-    // TODO: Add migrate_sapling_to_orchard argument
```

Comment:
Perhaps I will learn why later in this review, but I don't currently know why this TODO is removed.